lib/Listener/RegisterFlowOperationsListener.php

```diff
@@ -33,6 +33,9 @@
 use OCP\WorkflowEngine\Events\RegisterOperationsEvent;
 use Psr\Container\ContainerInterface;
 
+/**
+ * @template-implements IEventListener<Event>
```

Comment:
Should be `@template-implements IEventListener<RegisterOperationsEvent>` ideally. (Psalm may complain about the instanceof check below being redundant, but it makes sure that psalm checks this gets registered as a listener only for the right event type)

Reply:
I see. Most usages seem to be not ideal then, not just here. Thanks for pointing it out.